web/docs/understand/language/frontends/vast.md

```diff
@@ -1,6 +1,6 @@
-# VASTQL
+# VAST
 
-The **VAST** **Q**uery **L**anguage (VASTQL) frontend implements the
+The **VAST Language** frontend implements the
```

Comment:
```suggestion
The VAST language frontend implements the
```

Reply:
Replaced "VAST Language" with "VAST language" in https://github.com/tenzir/vast/pull/2959/commits/4b289bb23fce3c786bad9b58305f8910e4d0182a